packages/policycoreutils/Cargo.toml

```diff
@@ -12,12 +12,12 @@ path = "../packages.rs"
 releases-url = "https://github.com/SELinuxProject/selinux/releases/"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/SELinuxProject/selinux/releases/download/3.7/policycoreutils-3.7.tar.gz"
-sha512 = "30e3413b15df0bf1a994d2b3a03a719f89b3ee521a708b92fcc684822152145722cb3ef28fd5b7c42b779281b0bd4d69d65c0bc2605eec1af3f388609d985500"
+url = "https://github.com/SELinuxProject/selinux/releases/download/3.8/policycoreutils-3.8.tar.gz"
```

Comment:
The full set of SELinux packages should always be updated together, not just one of them.